examples/00-EDB/psd_example.py

```diff
@@ -0,0 +1,144 @@
+# # PSD Example
+
+
+# This is an example notebook from
+# [matplotlib](https://matplotlib.org/stable/gallery/lines_bars_and_markers/psd_demo.html#sphx-glr-gallery-lines-bars-and-markers-psd-demo-py)
+
+
+# # Power spectral density (PSD)
+#
+# Plotting power spectral density (PSD) using ``psd``.
+#
+# The PSD is a common plot in the field of signal processing. 
+# NumPy has many useful libraries for computing a PSD. Below we demo a few examples of 
+# how this can be accomplished and visualized with Matplotlib.
```

Comment:
```suggestion
# NumPy has many useful libraries for computing a PSD. The following code provides a few examples of 
# how this can be accomplished and visualized with Matplotlib.
```
Avoid "we" and page positions (like below and above).